### ISSUE TITLE
StreamDeck has to be reloaded every time an action is moved.

### ISSUE THREAD

**Issue by icessassin** [levindecaro/streamdeck-ytmd#10]
Actions will load data on restart but not if moved, when moved back to the original location they show the correct data otherwise show Waiting for data...
You probably already know this, im just happy to have the plugin back. Nice work getting it working with the new version.

**Comment by levindecaro** [comment]
@icessassin yes, need some workaround for this, it is due to limitation in current YTMD, in short, if the player is stopped, it won't tell the plugin the status, if the plugin actively probe YTMD, YTMD has rate limit will block the plugin. Seems hard to balance this situation. Will figure it out sometime, not really a big deal right?

**Comment by icessassin** [comment]
Not a big deal at all. Once your decks set up you dont normally move things
around anyway.
